sdks/python/apache_beam/testing/analyzers/perf_analysis_test.py

```diff
@@ -193,6 +198,32 @@ def test_alert_on_data_with_reported_change_point(self, *args):
         big_query_metrics_fetcher=None)
     self.assertFalse(is_alert)
 
+  def test_change_point_has_anomaly_marker_in_gh_description(self):
+    metric_values, timestamps = get_fake_data_with_change_point()
```

Comment:
fake data returns a series with a changepoint exactly in the middle. can we add a skew?

Reply:
Done